src/temporal/t.rast.climatologies/t.rast.climatologies.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+
+################################################
+#
+# MODULE:       t.rast.climatologies
+# AUTHOR(S):    Luca Delucchi, Fondazione Edmund Mach
+# PURPOSE:      t.rast.climatologies calculate climatologies for space time raster dataset
+#
+# COPYRIGHT:        (C) 2023 by Luca Delucchi
+#
+#                This program is free software under the GNU General Public
+#                License (>=v2). Read the file COPYING that comes with GRASS
+#                for details.
+#
+################################################
+
+# %module
+# % description: Calculate climatologies in a space time raster dataset
+# % keyword: temporal
+# % keyword: raster
+# % keyword: aggregation
+# % keyword: series
+# % keyword: time
+# %end
+
+# %option G_OPT_STRDS_INPUT
+# % key: strds
```

Comment:
Why not using input as key like most t.rast.* modules?

Reply:
Done